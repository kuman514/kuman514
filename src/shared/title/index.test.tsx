import { render } from '@testing-library/react';

import Title from '.';

describe('Title', () => {
  it('should show as expected', () => {
    const { container } = render(
      <Title>If you win you are Maverick, if you lose you are Yasuo.</Title>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should show as expected 2', () => {
    const { container } = render(
      <Title customClassName="text-black">Yasuo Maverick</Title>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
