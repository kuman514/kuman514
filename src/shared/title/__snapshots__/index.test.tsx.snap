// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should show as expected 1`] = `
<h1
  class="text-bold text-6xl undefined"
>
  If you win you are Maverick, if you lose you are Yasuo.
</h1>
`;

exports[`Title should show as expected 2 1`] = `
<h1
  class="text-bold text-6xl text-black"
>
  Yasuo Maverick
</h1>
`;
